docs/source/020_advanced.rst

```diff
@@ -44,15 +44,12 @@ modulefile for git one does: ::
     $ cd git
     $ cat > 2.6.2.lua
     local home    = os.getenv("HOME")
-    local version = myModuleVersion() 
-    local pkgName = myModuleName() 
+    local version = myModuleVersion()
+    local pkgName = myModuleName()
     local pkg     = pathJoin(home,"pkg",pkgName,version,"bin")
     prepend_path("PATH", pkg)
-    ^D  
+    ^D
 
-Starting first from the name: git/2.6.2.lua, modulefiles with the .lua
-extension are assumed to be written in lua and files without are
-assumed to be written in TCL.
```

Comment:
This paragraph is duplicated immediately below. I wasn't sure if I should get rid of this one or the next one. Honestly, both locations kind of feel awkward.